src/components/layouts/Layout.js

```diff
@@ -8,6 +8,7 @@ import SEO from './Seo';
 import { defaultTheme } from '../styledComponents';
 
 const Layout = ({ children, routeSlug }) => {
+  // Makes pages starting on the top (0, 0) when pages are reloaded.
```

Comment:
"Scrolls back to the top of the page (0, 0) when reloaded."

(Also, is this what you mean to say? Does this effect get triggered on reload (e.g. cmd - R) or on navigation changes?)

Reply:
Yes that I meant on cmd - R